docs/compare-to-ipywidgets.rst

```diff
@@ -10,36 +10,36 @@ Performance
 
-The ipywidgets functions expect the entire output to be regenerated everytime the slider value changes. This means that
-these functions will work best with the inline backend. In fact they even have some `special casing <https://github.com/jupyter-widgets/ipywidgets/blob/6be18d9b75353f7b4a1c328c6ea06d8959f978f6/ipywidgets/widgets/interaction.py#L230>`_
-to better support the inline backend. Unfortunately this does not work well with the interacive ``ipympl`` backend which
-expects to be shown only once and then updated with Matplotlib methods as controls change. The result of this is that you will often end up making
-many new figures or recreating the the entire plot every time the sliders change values. 
+The ipywidgets functions expect the entire output to be regenerated every time a slider value changes. This means ipywidgets functions will work best with 
+the ``inline`` backend. In fact, some `special casing <https://github.com/jupyter-widgets/ipywidgets/blob/6be18d9b75353f7b4a1c328c6ea06d8959f978f6/ipywidgets/widgets/interaction.py#L230>`_
+is even available to better support the inline backend. Unfortunately this process does not work well with the interacive ``ipympl`` backend. The ``ipympl`` backend 
+expects to be shown only once, then updated with Matplotlib methods as controls change. What results is you needing to make
+multiple new figures---or recreate the entire plot---every time a slider value changes. 
 
-You can get around these performance issues by using the ``interact`` function and having the called function use the Matplotlib updating methods 
-such as `line.set_data`. Unfortunately you then end up needing to remember how to do this as well find that you are repeating yourself.
-This was the initial motivation for this library and brings us to the reason of convenience.
+It is possible to get around these performance issues by using the ``interact`` function and setting the called function to use Matplotlib updating methods 
+(such as ``line.set_data``). However in this case, not only do you need to remember how to do this, but over time you will find you are repeating yourself.
+Reducing these performance barriers was the initial motivation for the mpl_interactions library, and also brings us to the reason of convenience.
 
 Portability
 -----------
 
-mpl_interactions will make use of the widgets provided by ipywidgets if they are available, but unlike interactive output, it will
-work if called from a script or an (i)python repl by falling back to the builtin Matplotlib
-`widgets <https://matplotlib.org/api/widgets_api.html?highlight=widgets#module-matplotlib.widgets>`_
+mpl_interactions will make use of the widgets provided by ipywidgets if they are available. Unlike interactive output, it will still 
+work if called from a script or an (i)python reply by falling back to the built in Matplotlib
```

Comment:
This definitely looks like a mistake on my part but repl is actually a real term. I probably should have capitalized it though. It stands for Read, Evaulate, Print, Loop and basically means that it will read in your input, evaluate it, print the result and repeat ad infinitum 

https://ipython.readthedocs.io/en/stable/development/how_ipython_works.html?highlight=repl